semanticcpg/src/main/scala/io/shiftleft/semanticcpg/language/nodemethods/MethodMethods.scala

```diff
@@ -5,13 +5,21 @@ import io.shiftleft.semanticcpg.language._
 import io.shiftleft.semanticcpg.language.types.expressions.ControlStructure
 import io.shiftleft.semanticcpg.language.types.expressions.generalizations.AstNode
 import io.shiftleft.semanticcpg.language.types.structure.{Local, MethodParameter}
+import scala.jdk.CollectionConverters._
+import io.shiftleft.Implicits.JavaIteratorDeco
 
 class MethodMethods(val node: nodes.Method) extends AnyVal {
 
   def parameter: MethodParameter = node.start.parameter
 
   def methodReturn: nodes.MethodReturn = node.start.methodReturn.head
 
+  def formalReturn: nodes.MethodReturn =
```

Comment:
This is a duplicate to `methodReturn` which is 2 lines above.

Reply:
It is indeed. But I'm guessing you like this one more, since it's defined on the node?
295505b deletes the old `methodReturn` which forwards to the DSL implementation, renames this to `methodReturn` and changes the DSL to use the node-implementation - looks good?

Reply:
Yes looks good.